caom2utils/caom2utils/blueprints.py

```diff
@@ -140,45 +142,37 @@ class ObsBlueprint:
         'Observation.metaReadGroups',
         'Observation.metaProducer',
         'Observation.requirements.flag',
-
```

Comment:
This would be an example where by automatic formatting makes the code less readable.

Reply:
I can keep the 120 character line length changes, and undo the less readable bits, if you would prefer that? I can also go around and redo the strings.